app/packages/core/src/components/FieldLabelAndInfo/index.tsx

```diff
@@ -514,6 +516,14 @@ function toRenderValue([key, value]): [string, string] {
   switch (typeof value) {
     case "boolean":
       return [key, value ? "True" : "False"];
+    case "object":
+      if (value.$date) {
+        return [key, new Date(value.$date).toUTCString()];
```

Comment:
nice! I'd think local time makes sense for the user looking at it in the UI. cc @brimoor 
```.toLocaleString()```

Reply:
When rendering `DateTimeField`s in the sidebar, the App respects the user's timezone setting: 
https://docs.voxel51.com/user_guide/config.html#configuring-a-timezone

It would be nice to respect that in the field tooltip too.